go/vt/vttablet/tabletserver/throttle/config/mysql_config.go

```diff
@@ -0,0 +1,103 @@
+/*
+ Copyright 2017 GitHub Inc.
+
+ Licensed under MIT License. See https://github.com/github/freno/blob/master/LICENSE
+*/
+
+package config
+
+//
+// MySQL-specific configuration
+//
+
+// DefaultMySQLPort is the default mysql port
+const DefaultMySQLPort = 3306
```

Comment:
AFAIK, this default port will most likely not work with Vitess. If that is the case, should we even have a default port?

Reply:
it's a bit moot in our case. That's because we get the port from the tablet, and it's guaranteed that the tablet has the port. The constant `DefaultMySQLPort` is derived from the original `freno`. I'm not sure whether I should cut off anything that's unrelated to `vitess` or just adapt the essentials. The more we diverge from `freno` (and we have already diverged a lot), the harder it will become to compare against upstream and get updates if needed.

Reply:
This was giving incorrect impression, so I pointed out. If we want to keep this code sync with upstream freno, may be it is ok to keep.

Reply:
Removed `DefaultMySQLPort`